app/javascript/src/awards.js

```diff
@@ -7,18 +7,51 @@ document.addEventListener("turbolinks:load", () => {
       post_id: postId,
       award_id: awardId,
     };
-    
+
     $.ajax({
       type: "POST",
       url: "/reward/",
       data: postData,
-      success: function(){
-        $('#post-awards').append(`<img src="${self.src}" class="post-award"/>`);
-        $("#awardsSelectModal").modal("hide");
+      success: function(response){
+        if (response["bought"]) {
+          $('#post-awards').append(`<img src="${self.src}" class="post-award"/>`);
+          $("#awardsSelectModal").modal("hide");
+        }
       },
       error: function(){
         console.error("Error occured in reward process!");
       }
     });
   });
+
+  var originalValue;
```

Comment:
Ieliec šo mainigo iekš funkcijas body. Viņu vairs nekur neizmanto

Reply:
partaisiju par "let" nevis "var' lai nav tik globals, bet likt funkcija nevaru since vinu izmanto 2 atskirigas funckijas lai nodotu datus, del kuram ari es vinu uztaisiju.